reach

```diff
@@ -115,10 +115,143 @@ run_s () {
   sh -ac "$TMP/out.sh" "$0"
 }
 
+gen_reachc_id () {
+  REACH_ENV_DIR="${XDG_CONFIG_HOME:-$HOME/.config}/reach/"
+  REACH_ENV_FILE="${REACH_ENV_DIR}/env"
+  MD5_CMD=md5sum||md5
```

Comment:
Does this really work for you? It doesn't for me.

Reply:
It did on a test I ran on my macbook.

But I changed this to a more concise approach.
Thanks for testing and flagging this out.